axon-server-connector/src/main/java/org/axonframework/axonserver/connector/AxonServerConnectionManager.java

```diff
@@ -98,7 +116,7 @@ public synchronized Channel getChannel() {
                     startInstructionStream(clusterInfo.getPrimary().getNodeName());
                     unavailable = false;
                     logger.info("Re-subscribing commands and queries");
-                    reconnectListeners.forEach(Runnable::run);
+                    scheduler.schedule(() -> reconnectListeners.forEach(Runnable::run), 100, TimeUnit.MILLISECONDS);
```

Comment:
Now that we're using a scheduler, wouldn't it be better to schedule each Runnable as its own scheduled task? This also reduces the risk of tasks not being executed when another throws an exception.

Reply:
Done